src/renderer/pages/Operations/components/Details.tsx

```diff
@@ -62,6 +62,11 @@ const Details = ({ tx, account, connection, signatory }: Props) => {
 
   const hasSender = isXcmTransaction(tx.transaction) || isTransferTransaction(tx.transaction);
 
+  const isDidverVisible =
```

Comment:
Maybe name it little bit specifier ?  but condition looks widely too

Reply:
Thought about that but can't find right words to describe that condition

Reply:
IN that case we can leave it as is